src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSourceProvider.cs

```diff
@@ -29,6 +29,7 @@ namespace Microsoft.CodeAnalysis.Editor.Implementation.Suggestions
     [Export(typeof(SuggestedActionsSourceProvider))]
     [ContentType(ContentTypeNames.RoslynContentType)]
     [ContentType(ContentTypeNames.XamlContentType)]
+    [ContentType("text")]
```

Comment:
This change would need to be reverted as this will lead to Roslyn binaries getting loaded for text files in non-Roslyn projects. I am going to work with the Editor team to figure out a work around for this.